webpack/webpack.dev.js

```diff
@@ -0,0 +1,6 @@
+const common = require('./webpack.common.js');
+
+module.exports = {
```

Comment:
There's a package called `webpack-merge`. Please research that and make use of it.